python/Pipfile

```diff
@@ -8,4 +8,4 @@ name = "pypi"
 [dev-packages]
 
 [requires]
-python_version = "3.8"
+python_version = "3.9"
```

Comment:
We'll have issues with 3.9 soon as well. Newer version is unavailable for amazon-linux, so we need extra actions to intall it there.